### PR TITLE
Update ConnectivityAppDefinitions.Codeunit.al

### PR DESCRIPTION
**Pull request by lukasaus**
App Id and URLs have been changed accordingly.
Spain was added to approved country list.<br /><br />Internal work item: AB#492553

### CODE REVIEW

File: Apps/W1/ConnectivityApps/app/src/Codeunits/ConnectivityAppDefinitions.Codeunit.al [hunk 1]
```diff
@@ -299,21 +299,22 @@ codeunit 20352 "Connectivity App Definitions"
...
         AppName := 'Bankfeed - Bank Statement Import & Reconciliation';
         AppPublisher := 'Softera Baltic';
         AppDescription := 'Bank statement import & reconciliation.';
-        AppProviderSupportURL := 'https://www.softera.lt/en/supported-banks-softera-bankfeed/';
-        AppSourceUrl := 'https://appsource.microsoft.com/en-us/product/dynamics-365-business-central/PUBID.softera_baltic%7CAID.softeradokubank%7CPAPPID.8051ee19-8246-489f-9f2c-4d89e1710f7e';
-        AppApprovedFor := 'LT,DK,HU,GB';
-        AppWorksOn := 'W1,DK,GB';
+        AppProviderSupportURL := 'https://bankfeed.com/banks/';
+        AppSourceUrl := 'https://appsource.microsoft.com/en-us/product/dynamics-365-business-central/PUBID.softera_baltic%7CAID.softeradokubank%7CPAPPID.74689c09-2ed3-4e69-a2e8-c9310a271b9a';
+        AppApprovedFor := 'LT,DK,HU,GB,ES';
```
Comment (sorenfriisalexandersen) [inline Apps/W1/ConnectivityApps/app/src/Codeunits/ConnectivityAppDefinitions.Codeunit.al:311]: We don't yet see enough customer uptake (3 tenants) in ES (Spain). 
We do see enough uptake in EE (Estonia). Could it be that you meant to add Estonia and not Spain?

Do you want to add Latvia (LV) as well, or are you not ready to do so yet? (It has enough uptake).

**Review by sorenfriisalexandersen — CHANGES_REQUESTED** [review]

File: Apps/W1/ConnectivityApps/app/src/Codeunits/ConnectivityAppDefinitions.Codeunit.al (hunk 1 above)
Comment (lukasaus) [reply Apps/W1/ConnectivityApps/app/src/Codeunits/ConnectivityAppDefinitions.Codeunit.al]: There must be some differences in customer uptake counting. For now, we removed Spain from the list. And will make another pull request after we figure out how to properly count the uptake.

But changes to AppId and URLs are important, so please confirm those. Thank you.

Comment (sorenfriisalexandersen) [reply Apps/W1/ConnectivityApps/app/src/Codeunits/ConnectivityAppDefinitions.Codeunit.al]: Thank you for the changes. Looks good now from a AppSource URL and country perspective.

**Comment by lukasaus** [comment]
Are there any more actions needed by me, or will it make the next minor release already?

**Comment by sorenfriisalexandersen** [comment]
We should be good here. @PredragMaricic can one from the team carry it through?

**Comment by JesperSchulz** [comment]
Availability update: We will publish a fix for this issue in the next update for release 23.

Build ID to track: 14264.

**Pull request closed without merging**